src/plugins/content_management/common/rpc/bulk_get.ts

```diff
@@ -5,31 +5,8 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-import { schema } from '@kbn/config-schema';
 import type { Version } from '@kbn/object-versioning';
-import { versionSchema } from './constants';
-import { GetResult, getResultSchema } from './get';
-
-import type { ProcedureSchemas } from './types';
-
-export const bulkGetSchemas: ProcedureSchemas = {
-  in: schema.object(
-    {
-      contentTypeId: schema.string(),
-      version: versionSchema,
-      ids: schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1 }),
-      options: schema.maybe(schema.object({}, { unknowns: 'allow' })),
-    },
-    { unknowns: 'forbid' }
-  ),
-  out: schema.object(
-    {
-      hits: schema.arrayOf(getResultSchema),
-      meta: schema.maybe(schema.object({}, { unknowns: 'allow' })),
-    },
-    { unknowns: 'forbid' }
-  ),
-};
+import { GetResult } from './get';
 
 export interface BulkGetIn<T extends string = string, Options extends void | object = object> {
```

Comment:
I could have moved the types to the server as well to keep the procedure definition all together in the same file. I just thought that imports would look weird (and force `import type` when importing from the `./server` directories while `./common` don't require that and can be imported with _plain_ `import`).

It would have triggered a lot of changes as well, making this PR harder to review.